### PR TITLE
[API]add experiment detail

### PR DESCRIPTION
**Pull request by fewdan**
### What problem does this PR solve? 
add experiment detail

### What is changed and how does it work?
add experiment detail

### Check List 

Tests 

 - No code

Code changes

 - Has Go code change

Side effects

 - NO

Related changes

 - NO

### Does this PR introduce a user-facing change?:

```release-note
NONE
```

### CODE REVIEW

File: pkg/apiserver/experiment/experiment.go [hunk 1]
```diff
@@ -500,8 +759,53 @@ func (s *Service) listExperiments(c *gin.Context) {
...
+		info, err = s.getIoChaosDetail(ns, name)
+	case "NetworkChaos":
+		info, err = s.getNetworkChaosDetail(ns, name)
+	case "TimeChaos":
+		info, err = s.getTimeChaosDetail(ns, name)
+	case "KernelChaos":
+		info, err = s.getKernelChaosDetail(ns, name)
+	case "StressChaos":
+		info, err = s.getStressChaosDetail(ns, name)
+	}
```
Comment (yeya24) [inline pkg/apiserver/experiment/experiment.go:808]: I prefer using `v1alpha1.KindXXX` here instead of plain string

Comment (fewdan) [reply pkg/apiserver/experiment/experiment.go:808]: OK, i will edit it.

**Pull request merged**